src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -2735,6 +2735,12 @@ internal static bool TryGetRelativeEntitySetPath(IEdmElement element, Collection
 
                     navigationProperties[navigationProperty] = new EdmPathExpression(paths);
 
+                    // In 7.4.1, FindNavigationTarget expected a binding path that included the path
+                    // to the contained entity set. In 7.4.2 FindNavigationTarget was fixed to work off
+                    // of the path from the contained entity set, but retained the old behavior as well
+                    // for backward compatibility. In the next breaking change we should remove that
+                    // behavior in FindNavigationTarget and remove this special handling of containsTarget
+                    // by always clearing the path.

```

Comment:
Do we have any items to track this task?

Reply:
Opened [https://github.com/OData/odata.net/issues/1114](https://github.com/OData/odata.net/issues/1114).

---
In reply to: [175176585](https://github.com/OData/odata.net/pull/1109#discussion_r175176585) [](ancestors = 175176585)